src/fuzzers/packet-fuzzer.cc

```diff
@@ -0,0 +1,50 @@
+#include "binpac.h"
+
+#include "Net.h"
+#include "Conn.h"
+#include "Sessions.h"
+#include "analyzer/Analyzer.h"
+#include "analyzer/Manager.h"
+#include "analyzer/protocol/pia/PIA.h"
+#include "analyzer/protocol/tcp/TCP.h"
```

Comment:
I think this include list could be chopped down to, based on the things you're using in the code.

```suggestion
#include "iosource/Packet.h"
#include "Event.h"
#include "Sessions.h"
```

Reply:
yes, thanks..  an oversight from starting with the pop3 one as a base